src/views/virtualmachines/list/VirtualMachinesList.tsx

```diff
@@ -45,24 +46,32 @@ import useVirtualMachineColumns from './hooks/useVirtualMachineColumns';
 import '@kubevirt-utils/styles/list-managment-group.scss';
 import './VirtualMachinesList.scss';
 
-type VirtualMachinesListProps = {
-  kind: string;
-  namespace: string;
-};
-
-const VirtualMachinesList: FC<VirtualMachinesListProps> = ({ kind, namespace }) => {
+const VirtualMachinesList: FC<ListPageProps> = ({
+  fieldSelector,
+  hideColumnManagement,
+  hideNameLabelFilters,
+  hideTextFilter,
+  kind,
+  nameFilter,
+  namespace,
+  selector,
+  showTitle,
+}) => {
   const { t } = useKubevirtTranslation();
   const catalogURL = `/k8s/ns/${namespace || DEFAULT_NAMESPACE}/catalog`;
   const { featureEnabled, loading: loadingFeatureProxy } = useFeatures(KUBEVIRT_APISERVER_PROXY);
   const isProxyPodAlive = useKubevirtDataPodHealth();
   const query = useQuery();
+
   const [vms, vmLoaded, loadError] = useKubevirtWatchResource<V1VirtualMachine[]>(
     {
+      fieldSelector,
```

Comment:
You need to check first with the proxy. Its probably working before merging; u need to run the proxy locally.

Reply:
In searchpage labels filtering is not working for VM list